src/settings-list.js

```diff
@@ -272,6 +272,25 @@ module.exports = [
         storageKey: 'disableWhispers'
     },
     {
+        name: 'Disable Frontpage Video',
+        description: 'Disable video on the frontpage',
+        default: false,
+        storageKey: 'disableFPVideo',
+        load: function() {
+            if (window.location.href === 'https://www.twitch.tv/' && bttv.settings.get('disableFPVideo') === true) {
+                $(window).load(function() {
```

Comment:
This is ran after the page loads: https://github.com/night/BetterTTV/blob/master/src/main.js#L311